nixos/tests/all-tests.nix

```diff
@@ -460,6 +460,7 @@ in {
   hound = handleTest ./hound.nix {};
   hub = handleTest ./git/hub.nix {};
   hydra = handleTest ./hydra {};
+  i18n = handleTest ./i18n.nix {};
```

Comment:
I would also favour `runTest` instead here.

Reply:
Comments like these are hard to deal with without further information. Why should `runTest` be used rather than `handleTest` in this case?

Reply:
I understand, I will add more description next time, thank you for reminding it to me.

Using `runTest` is actually newer and easier to write the test itself. I would advise to grep for `runTest` in `nixos/tests/` to have an idea. Pretty sure you'll adopt it.

Reply:
runTests is newer and handleTest is deprecated see #383870